src/instructlab/cli/model/train.py

```diff
@@ -240,6 +240,12 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     cls=clickext.ConfigOption,
     type=bool,
 )
+@click.option(
+    "--use-dolomite",
+    cls=clickext.ConfigOption,
+    config_sections=ADDITIONAL_ARGUMENTS,
+    type=bool,
```

Comment:
nit, but is there a `click.BOOL`? I know there are for other types

Reply:
yes, updated